rpc/jsonrpc/types/methods.go

```diff
@@ -1281,6 +1281,21 @@ type MixOutputCmd struct {
 	Outpoint string `json:"outpoint"`
 }
 
+// WalletPubPassphraseChangeCmd defines the walletpubpassphrasechange JSON-RPC command.
+type WalletPubPassphraseChangeCmd struct {
+	OldPassphrase string
+	NewPassphrase string
+}
+
+// NewWalletPubPassphraseChangeCmd returns a new instance which can be used to
+// issue a walletpubpassphrasechange JSON-RPC command.
+func NewWalletPubPassphraseChangeCmd(oldPassphrase, newPassphrase string) *WalletPubPassphraseChangeCmd {
```

Comment:
iirc, jrick said he doesn't want this. refer https://github.com/decred/dcrwallet/pull/1510#discussion_r316240839. 

I'm not sure this function is called from anywhere either, so you can just delete it.